src/shared/bindings.json

```diff
@@ -78,7 +78,7 @@
                     ]
                 },
                 {
-                    "name": "path",
+                    "name": "eventhubname",
```

Comment:
Is casing important here?  I thought they use `camelCasing` ? If its working that we are probaly good.

Reply:
It works with both I tested it again just to confirm. But I still used camel casing since it looks consistent and better and committed my new changes.